internal/version/version.go

```diff
@@ -0,0 +1,4 @@
+package version
+
+// Version is the current version of the Tekton MCP Server.
+const Version = "0.0.1"
```

Comment:
I think it should be a variable so that we can modify it using ldflags, see https://github.com/tektoncd/cli/blob/main/pkg/cmd/version/version.go#L40 for example